test/e2e/clusterctl_upgrade_test.go

```diff
@@ -35,16 +35,16 @@ var (
 	capvReleaseMarkerPrefix = "go://sigs.k8s.io/cluster-api-provider-vsphere@v%s"
 )
 
-var _ = Describe("When testing clusterctl upgrades using ClusterClass (CAPV 1.9=>current, CAPI 1.6=>1.7) [ClusterClass]", func() {
-	const specName = "clusterctl-upgrade-1.9-current" // prefix (clusterctl-upgrade) copied from CAPI
+var _ = Describe("When testing clusterctl upgrades using ClusterClass (CAPV 1.10=>current, CAPI 1.6=>1.7) [ClusterClass]", func() {
```

Comment:
```suggestion
var _ = Describe("When testing clusterctl upgrades using ClusterClass (CAPV 1.10=>current, CAPI 1.7=>1.7) [ClusterClass]", func() {
```

I think this should use 1.7 as it's the corresponding version to 1.10. We should bump this to 1.7=>1.8 as soon as first 1.8 versions are available.